src/lib/data/explore.ts

```diff
@@ -0,0 +1,52 @@
+export const exploreLinks: App.LinkList[] = [
+    {
```

Comment:
I did Audio and Games but the rest need to be added still.

Reply:
Sounds good, thanks for working on those and showing example. 

I'll add others asap